src/components/common/IdContainer.ts

```diff
@@ -1,16 +1,14 @@
-import _ from 'lodash/fp';
-import { ReactElement, useState } from 'react';
-
-export { useUniqueId } from '@terra-ui-packages/components';
+import { useUniqueId } from '@terra-ui-packages/components';
+import { ReactNode } from 'react';
 
 type IdContainerProps = {
-  children: (id: string) => ReactElement<any, any> | null;
+  children: (id: string) => ReactNode;
 };
 
 /**
  * DEPRECATED - should switch to useUniqueId pattern (below)
  */
-export const IdContainer = ({ children }: IdContainerProps) => {
-  const [id] = useState(() => _.uniqueId('element-'));
+export const IdContainer = ({ children }: IdContainerProps): ReactNode => {
```

Comment:
Was the idea of `useState` to keep the id unique between renders? I don't think it matters, but just curious.

Reply:
I was commenting on the fact that `useState` was removed. But now I realize `useUniqueId` itself takes care of this.